backend/migrations/00023_add_start_end_date_to_courses_table.sql

```diff
@@ -0,0 +1,11 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE public.courses ADD COLUMN start_dt date;
```

Comment:
this is going to need to rebase from main and this will have to be `migration #24` i believe

Reply:
rebased from main and made the migration sql file #24.